servers/zms/src/test/java/com/yahoo/athenz/zms/store/file/FileConnectionTest.java

```diff
@@ -603,4 +603,29 @@ public void testgetPendingMembershipApproverRoles() {
         }
     }
 
+    @Test
+    public void testProcessExpiredPendingMembers() {
+        File fileDir = new File("/home/athenz/zms_store");
+        File quotaDir = new File("/home/athenz/zms_quota");
+        try (FileConnection fileconnection = new FileConnection(fileDir, quotaDir)) {
+            try {
+                fileconnection.processExpiredPendingMembers(30, "sys.auth.monitor");
```

Comment:
we should have fail() after the call if we're expecting the call to throw an exception.